src/app/routes.tsx

```diff
@@ -59,7 +59,7 @@ const AppRoutes = (): React.ReactElement => {
   const nav = useNavigate();
   const location = useLocation();
 
-  axios.get('https://0.0.0.0:9443/api/v1/users/current/').catch(() => {
+  apiService.get('https://0.0.0.0:9443/api/v1/users/current/').catch(() => {
```

Comment:
Is the longer-term intent to eventually create functions in the `apiService` module and to move these long call arguments (like `get('https://0.0.0.0:9443/api/v1/users/current/')`) into them to abstract away their implementatino? In this case, something like `getCurrentUser`? Otherwise, I don't understand the benefit of simply replacing `axios` with `apiService`.

Reply:
yes, that was the plan. I abstracted it too much and too early and broke the application to a point I didn't know what was wrong, so I had to rollback on changes and retrace my steps. I am going to need @gitdallas to move forward with this, this refactor was way way out of my comfort zone. Any feedback is definetely welcome!

Reply:
rolled back on those changes, will open separate pr with new hook.